internal/commands/auditlog/export.go

```diff
@@ -0,0 +1,44 @@
+package auditlog
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/output"
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud/request"
+	"github.com/spf13/pflag"
+)
+
+var formats = []string{
+	request.ExportAuditLogFormatJSON,
+	request.ExportAuditLogFormatCSV,
+}
+
+// ExportCommand creates the "audit-log export" command
+func ExportCommand() commands.Command {
+	return &exportCommand{
+		BaseCommand: commands.New("export", "Export audit logs", "upctl audit-log export"),
+	}
+}
+
+type exportCommand struct {
+	*commands.BaseCommand
+	params request.ExportAuditLogRequest
+}
+
+func (c *exportCommand) InitCommand() {
+	fs := &pflag.FlagSet{}
+	fs.StringVar(&c.params.Format, "output", formats[0], fmt.Sprintf("Export format (typically %s). Note that this overrides the global --output flag.", strings.Join(formats, "|")))
```

Comment:
Overriding the global `--output` this way is a bit of a hack, but avoids adding another formatting option or adding awkward ones that won't work everywhere to the global.